test/gw-ci/CMakeLists.txt

```diff
@@ -54,12 +82,29 @@ set(TASK_LIST
   )
 add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
 
+# WCDA, low-res, ensemble da
+# -------------
+set(pslot "WCDA-hyb-C48mx500")
+set(letkf TRUE)
+set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C48mx500_hybAOWCDA.yaml)
+set(TASK_LIST
+  "gdasprepoceanobs"
+  "gdasmarineanalletkf"
+  "gdasmarinebmat"
+  "gdasocnanalprep"
+  "gdasocnanalrun"
+  "gdasocnanalchkpt"
+  "gdasocnanalpost"
+  )
+add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
+set(letkf FALSE)
+
 # Aero-Land DA, C96
 # -----------------
-set(pslot "Aero-Snow-3DVAR-C96")
-set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C96_atmaerosnowDA.yaml)
-set(TASK_LIST)  # empty list for now
-add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
+#set(pslot "Aero-Snow-3DVAR-C96")
+#set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C96_atmaerosnowDA.yaml)
```

Comment:
point to the new yaml name instead of commenting out

Reply:
C96C48_hybatmaerosnowDA.yaml

Reply:
done